arch/arm/boot/dts/qcom/msm8939.dtsi

```diff
@@ -27,7 +27,7 @@
 	qcom,msm-id = <239 0>, <241 0>, <263 0>;
 
 	chosen {
-		bootargs = "boot_cpus=0,4,5,6,7 sched_enable_hmp=1";
+		bootargs = "boot_cpus=0,1,2,3,4,5,6,7 sched_enable_hmp=1";
```

Comment:
why/why not? 

an ideal scenario would be using up all cpu's for boot, shutting them down later.

boot will eitherways cause battery drain (because...), so why not give it a boost?

disclaimer: haven't performed any definitive benchmarks yet.